ruby_event_store/spec/mappers/protobuf_spec.rb

```diff
@@ -150,50 +151,72 @@ module RubyEventStore
     specify 'defaults' do
       event = RubyEventStore::Proto.new(data: "One")
       expect(event.event_id).to match(/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}/)
-      expect(event.metadata).to eq({})
+      expect(event.metadata.to_h).to eq({})
       expect(event.data).to eq("One")
     end
+
+    specify 'metadata' do
+      event = RubyEventStore::Proto.new(data: nil, metadata: {one: 1})
+      expect(event.metadata[:one]).to eq(1)
+      expect do
+        event.metadata['doh']
+      end.to raise_error(ArgumentError)
+    end
   end
 
   module Mappers
     RSpec.describe Protobuf do
       let(:event_id)     { "f90b8848-e478-47fe-9b4a-9f2a1d53622b" }
+      let(:metadata) { {
+        one: 1,
+        two: 2.0,
+        three: true,
+        # four: Date.new(2018, 4, 17),
+        five: "five",
+        # six: Time.utc(2018, 12, 13, 11 ),
```

Comment:
Hmm could you shed some light on why these are commented out? Was there something in the gem that didn't work and so these types had to be skipped for now?

Reply:
No reason, I should uncomment them.

Reply:
Ok great ☺️ I was worried that these types didn't work for some reason, glad that it wasn't so